tutorials/remove all entity shadows

```diff
@@ -0,0 +1,327 @@
+**Add / Remove all shadows by code**
+
+by Outlandishly Crafted updated by Jeremy Benisek AKA CyberAxe
```

Comment:
Normally, we move credit into the footer. Do you mind moving this down to the bottom of the page?